src/framework/emqttb_worker.erl

```diff
@@ -164,6 +164,10 @@ connect(ConnOpstat, Properties0, CustomOptions, CustomTcpOptions, CustomSslOptio
             , {keepalive,    KeepAlive}
             ],
   {ok, Client} = emqtt:start_link(CustomOptions ++ Options),
+  %% dbg:tracer(),
```

Comment:
nit, maybe unintended?

Reply:
I left it here commented out because dbg has a strange syntax, so I don't want to figure it out every time.